datastore/datastore.go

```diff
@@ -72,6 +73,12 @@ type Query interface {
 	One(result interface{}) error
 	Sort(fields ...string) Query
 	Select(selector interface{}) Query
+	Count() (n int, err error)
```

Comment:
We don't need `Count` right? It won't give us an accurate count taking into the account the unique digests, so we'll need to use the aggregation to do the count as well? I'm fine adding it anyway, since it may be useful later, but probably prefer to add this in a separate PR.

Reply:
We don't need it now.